transformers/events/deny/converter.go

```diff
@@ -0,0 +1,53 @@
+package deny
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
+)
+
+type Converter struct {
+	LogNoteArgumentOffset int
+}
+
+func (c Converter) ToModels(_ string, logs []core.HeaderSyncLog, db *postgres.DB) ([]event.InsertionModel, error) {
+	numTopicsRequired := shared.ThreeTopicsRequired + c.LogNoteArgumentOffset
```

Comment:
The `LogNoteArgumentOffset` is so that we can use the same `ToModels` method for all deny methods right? And then just pass the offset for the Vat deny events?

Reply:
Yeah exactly. I considered implementing this field differently, like making it a boolean indicating whether the transformer is watching the `Vat` contract, but I felt like this way gave a more precise description of how the value is being used.